plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional, Union
+
+from hydra.core.config_store import ConfigStore
+
+
+@dataclass
+class DistributionConfig:
+    type: str
```

Comment:
could you add some comments here, ie what does `type` expect?

Reply:
I add the description of the properties of `DistributionConfig` including `type`. Please refer to the commit https://github.com/facebookresearch/hydra/pull/1132/commits/27299512157f6eecc308705e9a678a3c10990ee3 and https://github.com/facebookresearch/hydra/pull/1132/commits/69adb54d4c91755c581eb200d6ed1668043f5851.

Reply:
Thank you!
is this something we could potentially add Enum types for?

Reply:
Thank you for your suggestion. I introduced the Enum type `DistributionType` in https://github.com/facebookresearch/hydra/pull/1132/commits/c6e7c3206d2dd7a9aacf39f7c8c846d61aaf9ff1. Unlike `Direction`, I needed to convert `str` to `DistributionType` manually as follows:

```python
    kwargs = dict(config)
    if isinstance(config["type"], str):
        kwargs["type"] = DistributionType[config["type"]]
    param = DistributionConfig(**kwargs)
```

Please let me know if I overlooked something.